webcurator-core/src/main/java/org/webcurator/domain/model/simple/SimpleTargetGroup.java

```diff
@@ -0,0 +1,194 @@
+package org.webcurator.domain.model.simple;
+
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.webcurator.core.util.WctUtils;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+
+/**
+ * A TargetGroup contains a number of child targets or target groups.
+ */
+// TODO lazy="true"
+@Entity
+@Table(name = "TARGET_GROUP")
+@PrimaryKeyJoinColumn(name = "TG_AT_OID", referencedColumnName = "AT_OID")
+public class SimpleTargetGroup extends SimpleAbstractTarget {
+
+    /** The maximum length of the type field. */
+    public static final int MAX_TYPE_LENGTH = 255;
```

Comment:
Maybe it's better to use the constants from the non-simple variant to avoid duplication of information?